ehrapy/preprocessing/_imputation.py

```diff
@@ -616,27 +553,21 @@ def _warn_imputation_threshold(adata: AnnData, var_names: Iterable[str] | None,
     return var_name_to_pct
 
 
-def _get_non_numerical_column_indices(X: np.ndarray) -> set:
+def _get_non_numerical_column_indices(x: np.ndarray) -> set:
```

Comment:
It used to be called `X` because it was designed to be used for the `X` matrix of AnnData. For such a general function though, it's a terrible name.

```suggestion
def _get_non_numerical_column_indices(arr: np.ndarray) -> set:
```